### PR TITLE
Added utils

### PR DESCRIPTION
**Pull request by rschwabco**
Adds some useful utility functions

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    );
+
+    return true;
+  } catch (e) {
+    throw new Error(`Error upserting vectors into index: ${e}`);
+  }
+};
+
+
+const utils = {
```
Comment (jhamon) [inline src/utils.ts:84]: Did you mean to include `waitUntilIndexIsReady` here?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pinecone-database/pinecone",
-  "version": "0.1.5",
```
Comment (jhamon) [inline package.json:3]: Can you revert this change? CI handles version bumps now.

**Review by jhamon — APPROVED** [review]
Just few small things noted in comments to fix + ship.

Also, if you have a little more time, I think there's a test helper very similar to `waitUntilIndexIsReady` that you could probably replace now.

File: package.json (hunk 2 above)
Comment (rschwabco) [reply package.json:3]: Sweet!!

File: src/utils.ts (hunk 1 above)
Comment (rschwabco) [reply src/utils.ts:84]: No, I thought that it's really mostly useful in the context of `createIndexIfNotExist`, but we could export it as well.

**Pull request merged**